README.md

```diff
@@ -1 +1,77 @@
-# ruby-capstone
\ No newline at end of file
+## OOP school library
+
+This app is develped in ruby and allows users to:
+
+- Ruby console app with an user interface.
+- You can add books or music albums, and you can create a label or genre for the said item.
+- The application preserves user data by using JSON files while exiting the application. 
+
+## Built With
+
+- Ruby
+- OOP
+
+### Technologies used
+
+- Github
+- GitFlow
+- Visual Studio Code
+- rvm
+- Rspec
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- go to the directory that you want to clone the repository in.
+- right click and choose Git Bash here.
```

Comment:
- [ ] Also, you need to make a presentation video where you are:

1. presenting the features of the project.
1. The maximum length of the video is 5 minutes.
1. The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
1. All the members in the video should appear in the video and talk about the project.
1. The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

Reply:
We added a video description in Pull request